metaflow/plugins/kfp/metaflow_base_image_creation/Dockerfile

```diff
@@ -54,3 +58,7 @@ RUN echo "%users ALL=(ALL) NOPASSWD:/usr/local/bin/pip" >> /etc/sudoers && \
     echo "%users ALL=(ALL) NOPASSWD:/usr/local/bin/python" >> /etc/sudoers && \
     echo "%users ALL=(ALL) NOPASSWD:/usr/bin/apt-get" >> /etc/sudoers && \
     echo "%users ALL=(ALL) NOPASSWD:/usr/bin/apt" >> /etc/sudoers
+
+# Copy exit_handler, get_workflow_uid, and s3_sensor code to be used
+# by the Metaflow codebase.
+COPY ./metaflow_container_op /opt/metaflow/metaflow_container_op
```

Comment:
I believe this change should be a separate PR and task (that of putting the `exit_handler, get_workflow_uid, and s3_sensor` in the image).

This is a bigger change than it seems, because how do we version and tag and always update this image now that code is being copied to it?  And ensure that the versions of metaflow code and the image match?